src/tests/ftest/scrubber/performance.yaml

```diff
@@ -0,0 +1,54 @@
+hosts:
+  test_servers: 2
+  test_clients: 2
+timeout: 660
+setup:
+  start_servers_once: false
+server_config:
+  name: daos_server
+  engines_per_host: 2
+  engines:
+    0:
+      pinned_numa_node: 0
+      nr_xs_helpers: 1
+      fabric_iface: ib0
+      fabric_iface_port: 31416
+      log_file: daos_server0.log
+      log_mask: DEBUG,MEM=ERR
+      env_vars:
+        - DD_MASK=mgmt,md,rebuild
+      storage: auto
+    1:
+      pinned_numa_node: 1
+      nr_xs_helpers: 1
+      fabric_iface: ib1
+      fabric_iface_port: 31516
+      log_file: daos_server1.log
+      log_mask: DEBUG,MEM=ERR
+      env_vars:
+        - DD_MASK=mgmt,md,rebuild
+      storage: auto
+pool:
+  size: 95%
+  rebuild_timeout: 120
+  pool_query_timeout: 30
+container:
+  type: POSIX
+  control_method: daos
+  properties: "cksum:crc16"
+ior:
+  ior_timeout: 60
+  client_processes:
+    ppn: 32
+  test_file: daos:testFile
+  dfs_destroy: false
+  write_flags: "-w -C -e -g -G 27 -k -Q 1 -v"
+  read_flags: "-r -R -C -e -g -G 27 -k -Q 1 -v"
+  api: DFS
+  block_size: 150G
+  dfs_oclass: RP_2G1
```

Comment:
FYI, IOR SSF performance is better with GX since it uses all targets

Reply:
Done.